tests/test_janitor.py

```
@@ -21,24 +21,32 @@
 
 
 @pytest.mark.parametrize("version", (VERSION, 10, "10"))
-def test_version_cast(version: Any) -> None:
-    """Test that version is cast to Version object."""
-    janitor = DatabaseJanitor(user="user", host="host", port="1234", dbname="database_name", version=version)
+def test_version_is_deprecated(version: Any) -> None:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `version`

(ANN401)